deploy-manage/deploy/elastic-cloud/manage-deployments.md

```diff
@@ -1,17 +1,22 @@
 ---
+applies_to:
+  deployment:
+    ess: ga
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-manage-deployment.html
 ---
 
 # Manage deployments [ec-manage-deployment]
```

Comment:
I think this page could be restructured to be more intentional about the order of things. something like [this shape](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/499/deploy-manage/deploy/cloud-enterprise/deploy-an-orchestrator)